doc/source/serve/production-guide/fault-tolerance.md

```diff
@@ -0,0 +1,304 @@
+
+::::{tabbed} Vanilla Config
+```yaml
+...
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-sample
+spec:
+...
+```
+::::
+
+::::{tabbed} Fault Tolerant Config
+:selected:
+```yaml
+...
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-sample
+  annotations:
+    ray.io/ft-enabled: "true"
+    ray.io/external-storage-namespace: "my-raycluster-storage-namespace"
+spec:
+...
+```
+::::
+
+The annotations are:
+* `ray.io/ft-enabled` (REQUIRED): Enables GCS fault tolerance when true
+* `ray.io/external-storage-namespace` (OPTIONAL): Sets the [external storage namespace]
+
+Next, you need to add the `RAY_REDIS_ADDRESS` environment variable to the `headGroupSpec`:
+
+::::{tabbed} Vanilla Config
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+    ...
+spec:
+    ...
+    rayClusterConfig:
+        headGroupSpec:
+            ...
+            template:
+                ...
+                spec:
+                    ...
+                    env:
+                        ...
+```
+::::
+
+::::{tabbed} Fault Tolerant Config
+:selected:
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+    ...
+spec:
+    ...
+    rayClusterConfig:
+        headGroupSpec:
+            ...
+            template:
+                ...
+                spec:
+                    ...
+                    env:
+                        ...
+                        - name: RAY_REDIS_ADDRESS
+                          value: redis:6379
+```
+::::
+
+`RAY_REDIS_ADDRESS`'s value should include the Redis deployment name (e.g. `Redis`) and its port (e.g. `6379`). This example matches last section's [example config](one-node-redis-example).
+
+You can also expose your Redis cluster's port, so it can be accessed from outside the cluster as well:
+
+::::{tabbed} Vanilla Config
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+    ...
+spec:
+    ...
+    rayClusterConfig:
+        headGroupSpec:
+            ...
+            template:
+                ...
+                spec:
+                    ...
+                    ports:
+                        ...
+```
+::::
+
+::::{tabbed} Fault Tolerant Config
+:selected:
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+    ...
+spec:
+    ...
+    rayClusterConfig:
+        headGroupSpec:
+            ...
+            template:
+                ...
+                spec:
+                    ...
+                    ports:
+                        ...
+                        - containerPort: 6379
+                          name: redis
+```
+::::
+
+After you apply the Redis objects along with your updated `RayService`, your Ray cluster can recover from head node crashes without restarting all the workers!
+
+:::{seealso}
+Check out the KubeRay guide on [GCS fault tolerance](https://ray-project.github.io/kuberay/guidance/gcs-ft/#ray-gcs-fault-tolerancegcs-ft-experimental) for more details.
```

Comment:
Can we say what kinds of other details are available at this link?  Since it looks like there's already a lot of details given on the current page

Reply:
Sounds good, I added some details.